Packages/manifest.json

```diff
@@ -1,13 +1,12 @@
 {
   "dependencies": {
-    "com.unity.ads": "2.3.1",
-    "com.unity.analytics": "3.2.2",
-    "com.unity.collab-proxy": "1.2.15",
-    "com.unity.package-manager-ui": "2.0.3",
```

Comment:
while here should we update to latest LTS?

Reply:
i generally prefer to do those changes as a separate PR.